Storage/SQL/SQLitePasswords.swift

```diff
@@ -3,16 +3,54 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import Foundation
+import Shared
+import XCGLogger
 
-private class PasswordsTable<T>: GenericTable<Password> {
+private let log = XCGLogger.defaultInstance()
+
+class PasswordsTable<T>: GenericTable<Password> {
     override var name: String { return "logins" }
-    override var version: Int { return 1 }
+    override var version: Int { return 2 }
+    var encryption: EncryptionType = EncryptionType.AES256
+
+    private func secretKeyFor(key: String) -> String? {
```

Comment:
Why is this argument called `key` and used as an unencrypted Keychain item name? Should this be called `keyName` instead?